packages/strapi-plugin-i18n/admin/src/utils/sanitizeCTBContentTypeSchema.js

```diff
@@ -0,0 +1,59 @@
+import { has, get, omit } from 'lodash';
+import LOCALIZED_FIELDS from './localizedFields';
+
+const localizedPath = ['pluginOptions', 'i18n', 'localized'];
+
+const sanitizeCTBContentTypeSchema = (nextSchema, prevSchema) => {
```

Comment:
I'm not sure sanitizing is a valid term in this situation 🤔 . For me, sanitizing is about transforming invalid data and make sure they are valid.

For instance, when a user is able to fill some data, that data can't be trusted (for security reasons for example), we can eventually say that this data can be "dirty". Sanitizing, for me, is the process of making sure this potentially dirty data can't affect our system.

I'm maybe missing something, but in this scenario, I'm feeling this is more a transformation on the schema to make it have the shape we want it to have no?

Reply:
I agree the naming is incorrect, I was struggling to find a good naming, what about `mutateCTBContentTypeSchema`?

Reply:
Yes, this sounds better to me. Also, maybe defining what is the "type" of the input and the "type" of the output would help finding a great name; WYT? 😊